docs/source/compression/compression_config_list.rst

```diff
@@ -1,11 +1,190 @@
+
+A list of fully-qualified name of modules that will be excluded.
+
+exclude_op_names_re
+^^^^^^^^^^^^^^^^^^^
+
+A list of regular expressions for matching module names.
+The matched modules will be removed from the modules that need to be compressed.
+
+target_names
+^^^^^^^^^^^^
+
+A list of legal compressed target name, i.e., usually ``_input_``, ``weight``, ``bias``, ``_output_`` are support to be compressed.
+
+Two kinds of target are supported by design, module inputs/outputs(should be a tensor), module parameters/buffers:
+
+- Inputs/Outputs: If the module inputs or outputs is a singal tensor, directly set ``_input_`` for input and ``_output_`` for output.
+  ``_input_{position_index}`` or ``_input_{arg_name}`` can be used to specify the input target,
+  i.e., for a forward function ``def forward(self, x: Tensor, y: Tensor, z: Any): ...``, ``_input_0`` or ``_input_x`` can be used to specify ``x`` to be compressed,
+  note that ``self`` will be ignored when count the position index.
+  Similarly, ``_output_{position_index}`` can be used to specify the output target if the output is a ``list/tuple``,
+  ``_output_{dict_key}`` can be used to specify the output target if the output is a ``dict``.
+- Parameters/Buffers: Directly using the attribute name to specify the target, i.e., ``weight``, ``bias``.
+
+target_settings
+^^^^^^^^^^^^^^^
+
+A ``dict`` of target settings, the format is ``{target_name: setting}``. Target setting usually configure how to compress the target.
+
+All other keys(except these six common keys) in a config will seems as a shortcut of setting key, and will apply to all targets selected in this config.
+
+.. Note:: Each compression target can only be configure once, re-configuration will not take effect.
+
+Pruning Setting Keys
+--------------------
+
+sparse_ratio
+^^^^^^^^^^^^
+
+A float number between 0. ~ 1., the sparse ratio of the pruning target or the overall sparse ratio group of pruning tagrets.
+For example, if the sparse ratio is 0.8, and the pruning target is a Linear module weight, 80% weight value will be masked after pruning.
+
+max_sparse_ratio
+^^^^^^^^^^^^^^^^
+
+A float number between 0. ~ 1., for each pruning target, the sparse ratio after pruning will not be larger than this number,
+that means at most masked ``max_sparse_ratio`` pruning target value.
+
+min_sparse_ratio
+^^^^^^^^^^^^^^^^
+
+A float number between 0. ~ 1., for each pruning target, the sparse ratio after pruning will not be lower than this number,
+that means at least masked ``min_sparse_ratio`` pruning target value.
+
+sparse_threshold
+^^^^^^^^^^^^^^^^
+
+A float number, different from the ``sparse_ratio`` configure a specific sparsity, ``sparse_threshold`` usually used in some adaptive sparse cases.
+The value range is different for different pruning algorithms, please reference the pruner document to see how to configure it.
+In general, the higher the threshold, the higher the final sparsity.
+
+global_group_id
```

Comment:
`global_group_id` is jointly used with `sparse_ratio`? better to give a simple example.

Reply:
yes, add

```
``global_group_id`` should jointly used with ``sparse_ratio``.

For example, a model has three ``Linear`` modules (``'fc1'``, ``'fc2'``, ``'fc3'``),
and the expected total sparse ratio of these three modules is 0.5, then the config can be:

.. code-block:: python

    config = {
        'op_names': ['fc1', 'fc2', 'fc3'],
        'sparse_ratio': 0.5,
        'global_group_id': 'linear_group_1'
    }
```

Reply:
why should I write global_group_id here, where will it be used then?